src/python/sdk/examples/trade.py

```diff
@@ -0,0 +1,533 @@
+    else:
+        print(
+            f"  * There were {filled_size} limit orders filled by the orders placed."
+        )
+
+    report_best_price_levels(viewer, market_id)
+
+    input("\nPress enter to set-up and Account B with funds.")
+    account_B = setup_new_account(viewer, faucet_client, market_id)
+    print(f"Account B was set-up: {account_B.account_address}")
+
+    input("\nPress enter to place market orders (bid and ask) with Account B.")
+    place_market_order(Side.BID, account_B, market_id, 500)  # Buy 0.5 ETH
+    place_market_order(Side.ASK, account_B, market_id, 500)  # Sell 0.5 ETH
+    fill_size = len(
+        get_fill_events(viewer, account_B.account_address, market_id, 0)
+    )
+    print(f"Account B has finished placing 2 market orders.")
+    print(f"  * This resulted in {fill_size} limit orders getting filled.")
+
+    report_best_price_levels(viewer, market_id)
+
+    input("\nPress enter to cancel all of Account A's outstanding orders")
+    client_A = EconiaClient(NODE_URL, ECONIA_ADDR, account_A)
+    calldata1 = cancel_all_orders_user(ECONIA_ADDR, market_id, Side.ASK)
+    client_A.submit_tx_wait(calldata1)
+    calldata2 = cancel_all_orders_user(ECONIA_ADDR, market_id, Side.BID)
+    client_A.submit_tx_wait(calldata2)
+    cancel_size = len(
+        get_cancel_order_events(viewer, account_A.account_address, market_id, 0)
+    )
+    print(f"Account A has cancelled all {cancel_size} of their orders.")
+
+    report_best_price_levels(viewer, market_id)
+
+    input(
+        "\nPress enter to place competitive limit orders (top-of-book) with Account A."
+    )
+    _, start_ask_price = place_limit_orders_at_market(
+        viewer, account_A, market_id, 100, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    place_limit_orders_at_market(
+        viewer, account_A, market_id, 200, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    place_limit_orders_at_market(
+        viewer, account_A, market_id, 300, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    place_limit_orders_at_market(
+        viewer, account_A, market_id, 400, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    place_limit_orders_at_market(
+        viewer, account_A, market_id, 500, buy_ticks_per_lot, sell_ticks_per_lot
+    )
+    print("Account A has created multiple competitive limit orders!")
+
+    report_best_price_levels(viewer, market_id)
+
+    if start_ask_price == None:
+        exit()
+
+    input("\nPress enter to place spread-crossing limit orders with Account B.")
+    fills = get_fill_events(viewer, account_B.account_address, market_id, 0)
+    filled_size_pre = len(fills)
+    volume = 100 + 200 + 300 + 400 + 500
+    place_limit_order(Side.ASK, account_B, market_id, volume, buy_ticks_per_lot)
+    fills = get_fill_events(viewer, account_B.account_address, market_id, 0)
+    filled_size_ask = len(fills)
+    print(
+        f"  * There were {filled_size_ask-filled_size_pre} BID orders filled by the ASK order placement."
```

Comment:
Can this be samed something like `n_fills` instead of `fill_size`? I was confused at first because I thought it was saying the amount filled not the number of fills